codeql-custom-queries-ruby/example.ql

```diff
@@ -6,6 +6,7 @@
  */
 
 import ruby
+import codeql.ruby.AST
 
 from Block b
```

Comment:
This is another way of fixing the problem. The current solution has a spurious `import ruby` statement
```suggestion

from Ast::Block b
```